features/rhelah/atomic/sanity_test.feature

```diff
@@ -0,0 +1,106 @@
+@atomic @host_subscribed @ah_upgrade
+Feature: Atomic host sanity test
+    Describes the basic 'atomic host' command test
+
+Background: Atomic hosts are discovered
+      Given "all" hosts can be pinged
+
+  @pull_image
+  Scenario: 1. Pull latest image from repository
+      Given List all locally installed container images
+       When atomic update latest "centos" from repository
+       Then Check whether "centos" is installed
+
+  @mount_image_to_path
+  Scenario: 2. mount image to a specified directory
+       When atomic mount "image" to a specified "/mnt"
+       Then check whether atomic mount point "/var/mnt" exists
+
+  @unmount_image_from_path
+  Scenario: 3. unmount image previously mounted
+       When atomic unmount image from previous "/mnt"
+       Then check whether atomic mount point "/var/mnt" does not exist
+
+  @run_container_in_bg
+  Scenario: 4. docker run busybox with detach mode
+       When docker run "centos" detach mode with "top -b"
+       Then check whether there is a running container
+
+  @mount_container_to_path
+  Scenario: 5. mount running container to a specified directory
+       When atomic mount "container" to a specified "/mnt"
+       Then check whether atomic mount point "/var/mnt" exists
+
+  @unmount_container_from_path
+  Scenario: 6. unmount container previously mounted
+       When atomic unmount container from previous "/mnt"
+       Then check whether atomic mount point "/var/mnt" does not exist
+
+  @stop_container
+  Scenario: 7. atomic stop previous running container
+       When atomic stop container
+
+  @build_dangling_image
+  Scenario: 8. Build a new image from Dockerfile
+       When docker build an image from "https://raw.githubusercontent.com/projectatomic/atomic/master/tests/test-images/Dockerfile.1"
```

Comment:
@jlebon, yes, we have the same idea, ypu and I had ever considered to create a resources directory then put test related test files like Dockerfile.*, *.yaml, etc into it, I will make it happen in next patches.